<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elektra</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.res">
    <normaloff>:/window/elektra_icon.png</normaloff>:/window/elektra_icon.png</iconset>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="main_widget">
   <layout class="QVBoxLayout" name="verticalLayout_9"/>
  </widget>
  <widget class="QStatusBar" name="elektra_status"/>
  <widget class="QDockWidget" name="dock_settings">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string notr="true">Word Search Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dock_settings_widget">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-93</y>
          <width>287</width>
          <height>568</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="group_box_words">
           <property name="title">
            <string notr="true">Words</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QListWidget" name="list_words"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="button_add_word">
                <property name="text">
                 <string notr="true">Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button_remove_word">
                <property name="text">
                 <string notr="true">Remove</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="group_box_grid">
           <property name="title">
            <string>Puzzle</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Difficulty:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="combo_difficulty">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="currentIndex">
                 <number>2</number>
                </property>
                <item>
                 <property name="text">
                  <string>Right</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Right, down</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Right, down, with right-down diagonals</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Right, down, up, with right diagonals</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>All directions, with right diagonals</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>All directions, with all diagonals</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Grid size:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="combo_grid_size">
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <item>
                 <property name="text">
                  <string>7x7</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>8x8</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>9x9</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>10x10</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>11x11</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>12x12</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>13x13</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>14x14</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Cell size (in pixels):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="combo_box_size">
                <property name="currentIndex">
                 <number>3</number>
                </property>
                <item>
                 <property name="text">
                  <string>50x50</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>60x60</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>70x70</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>80x80</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>90x90</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>100x100</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Customize</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="check_capital">
              <property name="text">
               <string>Capital letters</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_shade">
              <property name="text">
               <string>White shades on cells</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_border">
              <property name="text">
               <string>Use borders</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_border_color">
              <property name="text">
               <string>Set border color</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="width_layout">
              <item>
               <widget class="QLabel" name="label_border">
                <property name="text">
                 <string>Border width:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="combo_border">
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <item>
                 <property name="text">
                  <string>1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>3</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>4</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>5</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Font size:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="combo_font_size">
                <property name="currentIndex">
                 <number>2</number>
                </property>
                <item>
                 <property name="text">
                  <string>20</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>30</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>40</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>50</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>60</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>70</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="button_font_path">
              <property name="text">
               <string>Change font path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_font_color">
              <property name="text">
               <string>Choose font color</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Generation</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Iterations:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_iterations">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>80</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="button_make_puzzle">
            <property name="text">
             <string>Make puzzle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_render_photo">
            <property name="text">
             <string>Render photo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_6">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string notr="true">Solutions</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Note that: All the words you input might not be here. Try use more iterations and enlarging the grid.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="list_solutions"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_5">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string notr="true">Export</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="button_export">
       <property name="text">
        <string>Export puzzle as an image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_export_sol">
       <property name="text">
        <string>Export solution to a text file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="about_button">
       <property name="text">
        <string>About Elektra</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.res"/>
 </resources>
 <connections/>
</ui>
